dunstctl

```diff
@@ -30,7 +30,7 @@ show_help() {
 	                                    notification with given ID
 	  history-rm ID                     Remove the notification from
 	                                    history with given ID.
-	  is-paused                         Check if pause level is greater than 0
+	  is-paused [-e]                    Check if pause level is greater than 0, optionally with exit code instead of text output
```

Comment:
```suggestion
	  is-paused [-e|--exit-code]        Check if pause level is greater than 0, optionally with exit code instead of text output
```
The long form should be documented as well.

Reply:
Ah, forgot that 👍